pkgs/top-level/all-packages.nix

```diff
@@ -10437,6 +10437,12 @@ with pkgs;
 
   seexpr = callPackage ../development/compilers/seexpr { };
 
+  semgrep = python3.pkgs.callPackage ../tools/security/semgrep {
+    inherit semgrep-core;
```

Comment:
Is the explicit `inherit` necessary?

Reply:
ah yeah. its already in all-packages so you're totally right.